csrc/anemone_mempool.c

```diff
@@ -48,6 +49,8 @@ void anemone_mempool_add_block (anemone_mempool_t *pool)
 // This happens when there is not enough space in the current block for the requested bytes.
 void * anemone_mempool_alloc_block (anemone_mempool_t *pool, size_t num_bytes)
 {
+  num_bytes = round_up_allocation(num_bytes);
```

Comment:
Yes, but `anemone_mempool_alloc_block` is a public function and could be called from *anywhere*.

Currently it may only be called from `anemone_mempool_alloc` but if its ever called from anywhere else we don't want it to mysteriously start breaking.